laba5_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include "student.h"
+#include "group.h"
+
+int main()
+{
+
+    int list_count;
+    cout << "how many students in group?\t";
+    cin >> list_count;
+    vector<Student> list;
+    for (int index = 0; index < list_count; index++)
+    {
+        list.push_back(Student());
+    }
+
+    for (int i = 0; i < list_count; i++)
+    {
+        int length;
+        cout << "name of student: ";
+        cin >> list[i].name;
+        cout << "how many marks " << list[i].name << " have?\t" << endl;
+        cin >> length;
+        list[i].marks_arr_create(length);
+        list[i].average_create();
+    }
+    cout << "____________________________________________________" << endl;
+    cout << "student name:\t"
+         << "student marks:\t"
+         << "mark average:\t" << endl;
+    cout << "____________________________________________________" << endl;
+
+    for (int i = 0; i < list_count; i++)
+    {
+        cout << list[i].name << " \t";
+        list[i].marks_output();
+        cout << list[i].average << endl;
+    }
+
+    Group ir_12;
+
+    float arr_average [list_count];
+    for (int i = 0; i < list_count; i++)
+    {
+
+        arr_average[i] = list[i].average;
```

Comment:
'arr_average[i] = list[i].average;'
pls rename "arr_average" variable to make it clear what it is for